runelite-client/src/test/java/net/runelite/client/plugins/itemstats/ItemStatEffectTest.java

```diff
@@ -79,6 +86,52 @@
 			.put(47, 25)  	.put(97, 40)
 			.put(49, 25)  	.put(99, 40)
 			.build();
+
+		// https://oldschool.runescape.wiki/w/Super_restore#Maximum_restoration
+		SUPER_RESTORE_TABLE = new ImmutableMap.Builder<Integer, Integer>()
+			.put(1, 8)    	.put(36, 17)  	.put(72, 26)
```

Comment:
Nit-pick, but tabs shouldn't be included after the initial indentation. All of the alignment of table entries should be done using spaces. For instance, this line would be changed as follows:
```suggestion
			.put(1, 8)      .put(36, 17)    .put(72, 26)
```